modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -59,3 +59,39 @@ chmod +x ./1-prepare.sh ./2-migrate.sh ./3-subscribe.sh ./4-wait.sh
 
 * Open {prod-short} dashboard and log in.
 
+[IMPORTANT]
+====
+If migration failed, use the following steps to rollback to previous {prod-short} installation.
+
+. Set the environment variables to use in the migration scripts:
++
+[source,bash,subs="+attributes"]
+----
+export INSTALLATION_NAMESPACE={prod-namespace} #<1>
+export PRODUCT_ID={prod-id}
+export PRODUCT_SHORT_ID={prod-id-short}
+export PRODUCT_DEPLOYMENT_NAME={prod-deployment}
+export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace}  #<1>
```

Comment:
for downstream should this be the old 2.15 namespace `openshift-workspaces` ? or the failed migration 3.0 namespace `openshift-devspaces` ? or the operator's namespace `openshift-operators` ?

Reply:
`openshift-workspaces` for 2.15.4
`openshift-operators` for 2.15.104

Reply:
so downstream we might want to make this explicit in the doc, or people will have a bad time.

Reply:
Sorry, I don't understand why.

Reply:
no, this should be `export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE=openshift-workspaces` according to updated PR description for the downstream values.

Reply:
That's basically the same as [1]. If it works why do I need to change anything?

[1] https://github.com/eclipse-che/che-docs/blob/1713753164ea2cadedc4f1691f22cfc389aaa9b4/modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc#L70-L81

Reply:
I'm saying I agree with your @tolusha, and that using `pre-migration-prod-checluster` is wrong. 

+1 to merge